src/core/ext/xds/xds_client_grpc.cc

```diff
@@ -167,7 +171,7 @@ class GrpcXdsClient::MetricsReporter final : public XdsMetricsReporter {
 
   void ReportServerFailure(absl::string_view xds_server) override {
     xds_client_.stats_plugin_group_.AddCounter(
-        kMetricServerFailure, 1, {xds_client_.key_, xds_server}, {});
+        kMetricServerFailure, uint64_t(1), {xds_client_.key_, xds_server}, {});
```

Comment:
Is this explicit type construction still needed?

Same question throughout.

Reply:
We need the static_cast somewhere otherwise the call to `AddCounter` is ambiguous, I can move the static_cast into the API though.